www/data/Annoucement.json

```diff
@@ -0,0 +1,8 @@
+{
+  "show": true,
```

Comment:
change to `false` for now since we won't use the announcement bar until there's an announcement ;-)